aspnetcore/blazor/hosting-models.md

```diff
@@ -22,10 +22,11 @@ The principal hosting model for Blazor is running client-side in the browser on
 
 ![Blazor client-side: The Blazor app runs on a UI thread inside the browser.](hosting-models/_static/client-side.png)
 
-To create a Blazor app using the client-side hosting model, use either of the following templates:
+To create a Blazor app using the client-side hosting model, use the following template:
 
-* **Blazor (client-side)** ([dotnet new blazor](/dotnet/core/tools/dotnet-new)) &ndash; Deployed as a set of static files.
-* **Blazor (ASP.NET Core Hosted)** ([dotnet new blazorhosted](/dotnet/core/tools/dotnet-new)) &ndash; Hosted by an ASP.NET Core server. The ASP.NET Core app serves the Blazor app to clients. The Blazor client-side app can interact with the server over the network using web API calls or [SignalR](xref:signalr/introduction).
+* **Blazor WebAssembly App** ([dotnet new blazorwasm](/dotnet/core/tools/dotnet-new)) &ndash; Deployed as a set of static files.
+
+In the above template, there is a **Hosted** option which configiures the application to use a ASP.NET Core server backend. The ASP.NET Core app serves the Blazor app to clients. The Blazor client-side app can interact with the server over the network using web API calls or [SignalR](xref:signalr/introduction).
```

Comment:
We can't say "above" ... they don't permit *directional* terms. You can say it like this tho ...

```suggestion
After selecting the **Blazor WebAssembly App** template, select the **Hosted** option to configure the app to use an ASP.NET Core server backend. The ASP.NET Core app serves the Blazor app to clients. The Blazor client-side app can interact with the server over the network using web API calls or [SignalR](xref:signalr/introduction).
```